tests/lib/schema_validation_test.py

```diff
@@ -178,6 +178,30 @@ def test_invalid_type(self, default_data):
             libnmstate.validator.validate(default_data)
 
 
+class TestIfaceMacAddress:
+    def test_valid_ethernet_mac_address(self, default_data):
```

Comment:
Can you also add invalid cases? Min, max and wrong characters should suffice. Strange that we did not have these already.

Reply:
BTW, you could use here parametrized https://docs.pytest.org/en/latest/parametrize.html

One test for valid entries and another for invalid ones.

Reply:
Done, thanks!